Rakefile

```diff
@@ -165,7 +165,7 @@ def push_release
 end
 
 def parse_tag
-  tag = ENV['CIRCLE_TAG']
+  tag = ENV['GITHUB_ACTIONS_TAG']
```

Comment:
Can you support both (e.g. `ENV['CIRCLE_TAG'] || ENV['GITHUB_ACTIONS_TAG']`) until the circle jobs are removed?

Reply:
Yes, I've added the CircleCI tag back in in the new commit.